spec.md

```diff
@@ -1076,6 +1076,36 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Enumerations
+
+Enumerations MUST follow the same guidelines as classes, except where otherwise noted here.
+
+Methods in Enumerations MUST follow the same guidelines as methods in classes.  Non-public methods MUST use `private`
+instead of `protected`.
+
+In the case of a backed enum, there MUST NOT be a space between the enum name and colon, and there MUST be exactly one
+space between the colon and the backing type.
+
+Enumeration cases MUST use CamelCase capitalization.
+
+Constants in Enumerations MAY use either CamelCase or UPPER_CASE capitalization.  CamelCase is RECOMMENDED,
+so that it is consistent with cases.
```

Comment:
Let's remove this and leave PSR-1 as the authoritative document for constant capitalization for now. In the future I think we want to consider rolling PSR-1 into this PER so that we can more effectively manage the content there.

Reply:
I don't think we should ignore it; case capitalization is the #1 question about enums I see people asking, moreso than anything else.  Yes, really. :smile:

Reply:
This comment is specific to the constant casing, I agree we should leave the case casing requirement

Reply:
I would still favor addressing both here, as they are, by design, closely related.

Reply:
What's meant by "with cases" here? What's "cases"? i.e. everything CamelCase or everything UPPER_CASE but not both? Maybe it worth stating it more explicitly.

Reply:
We should change that to "with case declarations": https://github.com/php-fig/per-coding-style/pull/7#discussion_r862201949